docs/howto/add_package_readme.md

```diff
@@ -81,3 +82,53 @@ List of placeholders that can be used in the Markdown templates:
       | data_stream.type | Data stream type. | constant_keyword |  |  |
       ...
       ```
+- `url <link_key>`: this placeholder is replaced by the URL defined in the `links_table.csv` file.
+    - Example of usage:
+      ```
+      Check {{ url "foo" }}
+      ```
+    - Let's assume that there exists a file `links_table.csv` with these contents:
+      ```
+      foo,http://url.com.test
+      help,http://other.url.com/help
+      ```
+    - Rendered output:
+      ```
+      Check http://url.com.test
+      ```
+    - Requirements:
+        - It is needed to define a file with the links definitions. More information [in this section](#requirements)
+- `url <link_key> <link_text>`: this placeholder is replaced by the URL defined in the `links_table.csv` file and it creates a
+  markdown link as `[link_text](url)`
+    - Example of usage:
+      ```
+      Check {{ url "help" "help guide" }}
+      ```
+    - Let's assume that there exists a file `links_table.csv` with these contents:
+      ```
+      foo,http://url.com.test
+      help,http://other.url.com/help
+      ```
+    - Rendered output:
+      ```
+      Check [help guide](http://other.url.com/help)
+      ```
+    - Requirements:
+        - It is needed to define a file with the links definitions. More information [in this section](#requirements)
+
+## Requirements
+
+### Links definitions file
+
+In order to be able to use `url` placeholder, links must be defined in a file.
+
+This file by default is located at the root of the repository with the name `links_table.csv`.
+It can be overwritten by setting the environment variable `ELASTIC_PACKAGE_LINKS_FILE_PATH` with
+the path to another file.
+
+The format of this file must be:
+```
+<key_1>,<url_1>
+<key_2>,<url_2>
+<key_3>,<url_3>
+```
```

Comment:
The only thing I'm not convinced about is the format of this file. I prefer YAML to the comma-separated form, but I'm fine if this is Elastic standard.

Reply:
This was a leftover in the docs, I've updated this file with the latest changes that I introduced to being this file a YAML. The format of the YAML would be:
```yaml
links:
  key_1: url_1
  key_2: url_2
```